app/assets/stylesheets/user.scss

```diff
@@ -4,16 +4,44 @@
 
 @import "ashby_settings.scss";
 
+.menu {
+  background-color: white;
+  border-radius: 8px;
+  height: 48px;
+}
+
+.menu p {
+  color: $ashby-gray;
+}
+
+.actions {
+  height: 250px;
```

Comment:
This is a strange style; it's rendering as a giant white patch. Is this the result of some copy-pasta?